client/src/components/RelatedItems/ProductCard.jsx

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ActionButton from './ActionButton.jsx';
 import noImageAvailable from './noImageAvailable.jpeg';
 
 function ProductCard({relatedProductInfo}) {
   const imageSource = relatedProductInfo.style.photos[0].url ? relatedProductInfo.style.photos[0].url : noImageAvailable;
+  const productprice = Number(relatedProductInfo.default_price);
   return (
-    <div className="card" style={{width: '20em'}}>
-      <img className="card-img-top" src={imageSource} style={{width: 'auto', height: '35vw', objectFit: 'cover'}} alt="Product image"></img>
-      <p>--- Action Button ---</p>
+    <div className="card border-dark" style={{width: '20em', margin: '1em', position: 'relative'}}>
```

Comment:
There are some options in bootstrap for setting position via classname: https://getbootstrap.com/docs/5.1/utilities/position/

There's also margins and sizing which is relative to parent element...so if, for example, you wanted to only display four cards within the carousel at a time, you could add margins/padding to each and set them to 20% width, etc. (https://getbootstrap.com/docs/5.1/utilities/sizing/) I know you have a lot to deal with already but in case you have time, consider refactoring via the bootstrap utilities!